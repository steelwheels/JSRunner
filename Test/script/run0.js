function main(args)
{
	console.log("main-func (b)") ;

	let proc = run("../Test/script/run0-func.js", stdin, stdout, stderr) ;
	proc.start(["a", "b"]) ;
	proc.waitUntilExit() ;

	console.log("main-func (e)") ;
}

