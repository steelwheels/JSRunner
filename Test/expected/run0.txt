main-func (b)
runc: a
main-func (e)
